# ID ящика, на который направлены данные
int8 can_id

# Массив направляемых данных
int8[8] can_data